contracts/EASResolver/EASResolver.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.7;
+
+import { SchemaResolver } from "./SchemaResolver.sol";
+
+import { IEAS, Attestation } from "./IEAS.sol";
+
+contract RecipientResolver is SchemaResolver {
+    mapping (address => bool) private whitelisted;

```

Comment:
So this resolver will be used for many DAOs. we can't let someone whitelisted on one dao to attest for someone in another dao. whitelisting should be per DAO.